### PR TITLE
push lists and tags updated

### PR DESCRIPTION
**Pull request by m4rc1e**

### CODE REVIEW

File: to_production.txt [hunk 1]
```diff
@@ -1,11 +1,19 @@
 # New
 ofl/briemhand # https://github.com/google/fonts/pull/7040
+ofl/freeman # https://github.com/google/fonts/pull/7548
+ofl/jacquard24charted # https://github.com/google/fonts/pull/7525
```
Comment (chrissimpkins) [inline to_production.txt:4]: Requires a Workspace menu family.

File: to_production.txt [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+ofl/jaini # https://github.com/google/fonts/pull/7208
+ofl/jainipurva # https://github.com/google/fonts/pull/7209
 ofl/jaro # https://github.com/google/fonts/pull/7391
+ofl/jersey25charted # https://github.com/google/fonts/pull/7536
```
Comment (chrissimpkins) [inline to_production.txt:8]: Requires a Workspace menu family

**Review by chrissimpkins — CHANGES_REQUESTED** [review]
Jacquard 25 Charted and Jersey 25 Charted require Workspace menu families.  I'll work with the Fonts eng team to land this before we merge these updates.  Should be by early next week.

**Comment by chrissimpkins** [comment]
> Jacquard 24 Charted and Jersey 25 Charted require Workspace menu families. I'll work with the Fonts eng team to land this before we merge these updates. Should be by early next week.

Changes opened and review requested from Nate.

**Comment by chrissimpkins** [comment]
CI is failing on Knowledge HTML tag checks

**Comment by chrissimpkins** [comment]
This should remain open? Or are you preparing a new set of lists?

File: to_production.txt (hunk 1 above)
Comment (chrissimpkins) [reply to_production.txt:4]: We continue to wait on the menu family update.  Not ready to merge yet.

File: to_production.txt (hunk 2 above)
Comment (chrissimpkins) [reply to_production.txt:8]: We continue to wait on the menu family update.  Not ready to merge yet.

**Comment by m4rc1e** [comment]
Sorry, Github automatically closed this PR when I merged #7613

File: to_production.txt (hunk 1 above)
Comment (chrissimpkins) [reply to_production.txt:4]: The menu family is updated in sandbox.  All clear here

File: to_production.txt (hunk 2 above)
Comment (chrissimpkins) [reply to_production.txt:8]: The menu family is updated in sandbox. All clear here

**Comment by chrissimpkins** [comment]
All set for merge from my standpoint.

**Pull request merged**